.travis.yml

```diff
@@ -1,46 +1,43 @@
 dist: bionic
 sudo: required
 language: minimal
-addons:
-  apt:
-    sources:
-    - sourceline: "ppa:chris-lea/redis-server"
-    packages:
-    - redis-server
-    - redis-tools
 
 branches:
   only:
   - master
 
 services:
 - docker
-- redis-server
 
-before_install:
-- sudo apt --yes --quiet update
+cache:
+  directories:
+  - docker_images
 
-# Use PostgreSQL 11
-- sudo apt --yes --quiet autoremove --purge libpq\* postgresql-{,client-,contrib-}{9,10}\*
-- sudo apt --yes --quiet install postgresql-11
+before_install:
+# Load saved Docker images from Travis' cache
+- gunzip --stdout docker_images/images.tar.gz | docker load || true
 
 install:
-- docker build --build-arg build_type=dev --tag azafea-tests .
-
-before_script:
-# Create the PostgreSQL role and database
-- sudo -u postgres createuser azafea
-- sudo -u postgres createdb --owner=azafea azafea-tests
+- docker build --build-arg build_type=dev --tag azafea-tests --pull .
 
-# Set the default passwords for PostgreSQL and Redis, as that's what the tests expect
-- sudo -u postgres psql --command="ALTER USER azafea WITH PASSWORD 'CHANGE ME!!';"
-- redis-cli config set requirepass 'CHANGE ME!!'
+before_cache:
+# Save Docker images to Travis' cache, but prune old ones first
+- mkdir -p docker_images
+- docker images --quiet --filter=dangling=true | xargs -r docker rmi
```

Comment:
Is this the same as `docker system prune`?

Reply:
Yeah, but this also works on older version of Docker. Doesn't matter anymore, I removed the whole cache thing (it made the build even slower).

Reply:
Ah ok, thanks for the tip about older Docker then. :slightly_smiling_face: